### PR TITLE
SDK Go generation for azure PKI

### PR DESCRIPTION
**Pull request by arpit2408**
Here is the approved API spec:- 

https://apiview.dev/Assemblies/Review/851e03d179094a33b4c0198fa30d5d5a

### CODE REVIEW

File: sdk/identity/azpki/cloud_config.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+)
+
+func init() {
+	cloud.AzureChina.Services[ServiceNamePKI] = cloud.ServiceConfiguration{
+		Audience: "https://api.loganalytics.azure.cn",
+		Endpoint: "https://api.loganalytics.azure.cn/v1",
+	}
+	cloud.AzureGovernment.Services[ServiceNamePKI] = cloud.ServiceConfiguration{
+		Audience: "https://api.loganalytics.us",
+		Endpoint: "https://api.loganalytics.us/v1",
```
Comment (chlowell) [inline sdk/identity/azpki/cloud_config.go:23]: Are these values correct? The client should send requests to log analytics endpoints?

File: sdk/identity/azpki/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Release History
+
+## 0.1.0 (2023-01-25)
```
Comment (chlowell) [inline sdk/identity/azpki/CHANGELOG.md:3]: This should be a future date. 2023-04-11 is the next scheduled release day

File: sdk/identity/azpki/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+Please find additional details for the API [here.](https://microsoft.sharepoint.com/:w:/t/PKIaaS/EVnYiASHsyBFjD72tLLv6VIBRIq1q5nPiitAC4apyNcTRw?e=FcvYsd&nav=eyJoIjoiMTcwMjg0MzMwNCJ9&wdOrigin=TEAMS-ELECTRON.p2p.bim&wdExp=TEAMS-TREATMENT&wdhostclicktime=1673615940893&web=1&xsdata=MDV8MDF8fDNhYjBlODhlOWYwMjQxZWRiZDBhMDhkYWZmZGY3ZWZifDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDB8MHw2MzgxMDM2MzI1OTk1NjM1MzB8VW5rbm93bnxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxNVFkzTkRjMk5qUTFPRGd4TkRzeE5qYzBOelkyTkRVNE9ERTBPekU1T2pBek5UUTFZMlkwTFRCaE9HTXRORE5rWWkwNFptWXdMVGd3WWprd1ptUmhZVEZsTWw4elptVTFOVFUwTnkweFltTTVMVFEzTTJJdE9UZ3dNeTB3TVRrMU1XVmpPVGM0T1RWQWRXNXhMbWRpYkM1emNHRmpaWE09fGFlMWI5MjE4NjdmNzQ2M2ZiZDBhMDhkYWZmZGY3ZWZifDA2NWFkYjU1Y2Y0MTQ5OGU4NTJhY2Y4MDhhYjk5YmYw&sdata=VjNrY2V4elJiYmxZdjhCWjNwV1FtZHNxT3VOZ1NTR0lFcmVwNnRHd2RJRT0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2Carpitsaxena%40microsoft.com&OR=Teams-HL&CT=1674780808458&clickparams=eyJBcHBOYW1lIjoiVGVhbXMtRGVza3RvcCIsIkFwcFZlcnNpb24iOiIyNy8yMzAxMDEwMDkxMSIsIkhhc0ZlZGVyYXRlZFVzZXIiOmZhbHNlfQ%3D%3D)
```
Comment (chlowell) [inline sdk/identity/azpki/README.md:9]: Is there a better link? Is this one accessible outside Microsoft?

**Review by chlowell — COMMENTED** [review]
Are you also adding tests?

File: sdk/identity/azpki/CHANGELOG.md (hunk 2 above)
Comment (benbp) [reply sdk/identity/azpki/CHANGELOG.md:3]: It should say unreleased

```suggestion
## 0.1.0 (Unreleased)
```

**Review by jhendrixMSFT — CHANGES_REQUESTED** [review]
These files aren't properly formatted.  Please execute `gofmt -w .` and commit the changes (this is also why CI is failing).

**Pull request closed without merging**